### PR TITLE
Documentation update for plot_top_losses to include Grad-CAM heatmap …

### PR DESCRIPTION
**Pull request by at110**
Documentation was updated for vision.learn to include Grad-Cam  heat-map option in plot_top_losses.

### CODE REVIEW

File: docs_src/vision.learner.ipynb [hunk 1]
```diff
@@ -514,8 +514,7 @@
    "cell_type": "markdown",
```
Comment (sgugger) [inline docs_src/vision.learner.ipynb]: <p>Can you just put backticks around the argument names? For instance `k`, `heatmap`. Then same for the functions `plot_top_losses` (without parenthesis) will then create a link to that function documentation during our conversion, so it's pretty neat.</p>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai/pull/1640/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

**Comment by sgugger** [comment]
A few minor changes requested. This is the first time we use ReviewNB for this, so let me know if you have any trouble seeing them.

**Comment by at110** [comment]
Thank for catching it. I can see your comments so ReviewNB seems to be working well. I added backpacks around function name. I excluded "()" from backticks as you suggested but saw other function names have "()" in backticks in document.

**Comment by sgugger** [comment]
Thanks!
For future reference, don't even put the parenthesis (I'll remove them, didn't want to bother you with this). If you catch other functions names in backticks with parenthesis, feel free to suggest a PR to remove them because it breaks the link toward them in the docs.

**Pull request merged**